src/pycram/process_module.py

```diff
@@ -40,14 +34,14 @@ def __init__(self, lock):
         """Create a new process module."""
         self._lock = lock
 
-    def _execute(self, designator: MotionDesignatorDescription.Motion) -> Any:
+    def _execute(self, designator) -> Any:
         """
         Helper method for internal usage only.
         This method is to be overwritten instead of the execute method.
         """
         pass
 
-    def execute(self, designator: MotionDesignatorDescription.Motion) -> Any:
```

Comment:
Why change the typing? Process Modules should still only accept MotionDesignators

Reply:
Yes, you are right. But importing the new BaseMotion in order to use it for type hinting leads to circular imports, since it is located in the motion_designator file and the motions themselves need the ProcessModuleManager. Removing the type hint is not fancy, but I didn't see an easy way to solve that problem